### DISCUSSION TITLE
How to change VpnHood server default port?

### DISCUSSION THREAD

**Discussion by devsynck** [vpnhood/VpnHood#16] [Q&A] [answered]
It seems VpnHood uses 0.0.0.0:443 by default. I am using VPS (Centos 7) which have Nginx already using 0.0.0.0:443. Is there any way to VpnHood along side webserver.

**Comment by trudyhood** [comment, marked as answer]
To change the server port check this:
[VpnHood Server Configuration](https://github.com/vpnhood/VpnHood/wiki/VpnHood-Server-Configuration)
